components/member-profile/index.js

```diff
@@ -5,10 +5,13 @@ import getBadges from './mock/get-badges';
 import classNames from './member-profile.module.scss';
 import ContributionType from './contribution-type/';
 import { motion } from 'framer-motion';
+import Tooltip from 'react-tooltip-lite';
 
 const renderBadgeImages = (badges) =>
   badges.map((badge) => (
-    <img src={badge.img} className={classNames.badge} alt={badge.title} key={badge.title} />
+    <Tooltip key={badge.title} content={badge.title} useDefaultStyles direction="down">
```

Comment:
@sridhar02 can we have the jsx elements in one function as we using badge too many times. You can take it in a function and return it. For usage of badhe too many times you can badge as an argument and destructe it inside the function. looks clean then

Reply:
@sridhar02